Week9Serialization/Week9Serialization/Functional Classes/Female.cs

```diff
@@ -0,0 +1,22 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Runtime.Serialization;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Week9Serialization
+{
+    /// <summary>
+    /// This class inherits from the Person class.
+    /// </summary>
+    [Serializable]
+    [DataContract]
+    public class Female: Person
+    {
+        public Female(String name = null)
```

Comment:
You'll typically want to comment constructors as well. A conventional description would be something like, "Initializes a new instance of the `<see cref="Female" />` class." The use of the see element allows navigation back to original class if/when XML comments are extracted into HTML documents using a 3rd party tool. It also allows  Visual Studio to update the name here if the class is renamed.